src/fragments/start/getting-started/flutter/add-api.mdx

```diff
@@ -22,9 +22,9 @@ On the creation form:
 
     ![Create app backend](/images/lib/getting-started/flutter/connect-to-cloud-create-app-backend.png)
 
-4. The on screen text should walk you through the deployment progress and, when the deployment status reads **Deployment completed**, click **Launch Studio**.
+4. The on-screen text should walk you through the deployment progress and, when the deployment status reads **Deployment completed**, click **Launch Studio**.
```

Comment:
I am a bit confused by the phrasing `The on-screen text should walk you through the deployment progress`, should this be `The on-screen text should walk you through the deployment process`?

Reply:
Thanks Caleb, I have updated it.